app/src/main/java/org/breezyweather/search/SearchActivity.kt

```diff
@@ -18,7 +18,6 @@ package org.breezyweather.search
 
 import android.content.Intent
 import android.os.Bundle
-import android.window.OnBackInvokedDispatcher
```

Comment:
This is inherited by the activity so it was unused. I can revert this change if it's something you want done here since it's not related to this particular pull request.